pages/workflow/PKPWorkflowHandler.inc.php

```diff
@@ -256,6 +257,33 @@ public function index($args, $request)
         $citationsForm = new PKP\components\forms\publication\PKPCitationsForm($latestPublicationApiUrl, $latestPublication);
         $publicationLicenseForm = new PKP\components\forms\publication\PKPPublicationLicenseForm($latestPublicationApiUrl, $locales, $latestPublication, $submissionContext, $authorUserGroups);
         $titleAbstractForm = new PKP\components\forms\publication\PKPTitleAbstractForm($latestPublicationApiUrl, $locales, $latestPublication);
+        $contributorForm = new PKP\components\forms\publication\PKPContributorForm($contributorApiUrl, $locales, $submissionContext);
+
+        $authorListGetParams = [
+            'contextIds' => $request->getContext()->getId(),
+            'count' => 30,
+        ];
+
+        $authorItems = [];
+        foreach ($latestPublication->getData('authors') as $contributor) {
+            $authorItems[] = Repo::author()->getSchemaMap()->map($contributor);
+        }
+
+        $authorCollector = Repo::author()->getCollector();
+        $authorCollector->filterByPublicationIds([$latestPublication->getId()]);
+        $authorItemsMax = Repo::author()->getCount($authorCollector->limit(null)->offset(null));
+        $contributorsListPanel = new \PKP\components\listPanels\PKPContributorsListPanel(
+            'contributors',
+            __('publication.contributors'),
+            [
+                'form' => $contributorForm,
+                'getParams' => $authorListGetParams,
+                'items' => $authorItems,
+                'itemsMax' => $authorItemsMax,
+                'publicationApiUrl' => $contributorPublicationApiUrl,
+                'canEditPublication' => $canEditPublication
+            ]
```

Comment:
Similar comments as above regarding `itemsMax` and `getParams`.

Reply:
`itemsMax` removed